### PR TITLE
YUNIKORN-233: Change default pagesize of datatable in yunikorn-web to 10

### PR DESCRIPTION
**Pull request by akhilpb001**

### CODE REVIEW

File: src/app/components/nodes-view/nodes-view.component.ts [hunk 1]
```diff
@@ -111,7 +111,7 @@ export class NodesViewComponent implements OnInit {
     }
   }
 
-  onPaginatorChanged(page: PageEvent) {
+  onPaginatorChanged() {
```
Comment (yangwwei) [inline src/app/components/nodes-view/nodes-view.component.ts:114]: hi @akhilpb001 

The rest of the changes are straightforward, but for this one. I just want to make sure this is intentional, is it?

Comment (akhilpb001) [reply src/app/components/nodes-view/nodes-view.component.ts:114]: Yes, its unused code, hence removed.

Comment (akhilpb001) [reply src/app/components/nodes-view/nodes-view.component.ts:114]: If we are getting travis precommit stage green, that means we are good to commit the code.

Comment (yangwwei) [reply src/app/components/nodes-view/nodes-view.component.ts:114]: Got it, thanks

**Pull request merged**